### DISCUSSION TITLE
Serve pages from different branch  from same repo ?

### DISCUSSION THREAD

**Discussion by viral1043** [community/community#21729] [Other Feature Feedback, Questions, & Ideas]
<p>Is there any way where we can have multiple pages in <em>different</em> branch and we can host it through git Hub pages from one repo.<br>
eg <br>
the documentation we want to show there will be stored as HTML in 2 branches<br>
master/docs and develop/docs</p>
<p>As per docs we can have it in master, master/docs or gh-pages branch.</p>
<p>But still want to confirm if there is any way to.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>If you’re asking if you can host two GitHub Pages sites from the same repository, then no. You can only host one GitHub Pages site per repository.</p>
<p>If my understanding is incorrect, can you give some more details as to what it is you’re trying to do?</p>
<p>Thanks!</p>

**Comment by fire-eggs** [comment]
<p>I’m pretty sure the request is (essentially) for links from different branches from a single repo.</p>
<p>E.g.</p>
<pre><code>&lt;a href="https://github.com/user/repo/raw/master/docs/doc1.html&gt;
</code></pre>
<p>and</p>
<pre><code>&lt;a href="https://github.com/user/repo/raw/develop/docs/doc1.html&gt;
</code></pre>
<p>where ‘master’ is the master branch, and ‘develop’ is another branch, both in the same repo.</p>
<p>I’ve not tried this myself or if there is some other link style that could be used.</p>
